chart/compass/values.yaml

```diff
@@ -109,20 +111,20 @@ global:
     # If secrets do not exist they will be created
     secrets:
       ca:
-        name: compass-connector-app-ca
-        namespace: compass-system
+        name: application-connector-certificate-overrides
+        namespace: compass-installer
```

Comment:
I do not think that it is correct.

Reply:
Should I revert it then? There's an override for this secret [here](https://github.com/kyma-incubator/compass/blob/96ceae69260e35234561995bcdb0c0b44ee182a7/installation/resources/installer-config-local.yaml.tpl#L4) if this changes anything.